docs/notebooks/nb_02_getting_started.ipynb

```diff
@@ -0,0 +1,208 @@
+{
```

Comment:
<p>A couple of edit suggestions:</p><p>1) Show how to specify the dynamic correlation solver and list all possible dynamic correlation solvers available</p><p>2) Same with the active space solver -&gt; list all possible active space solvers -&gt; ACI, CAS, FCI, etc</p><p>3) In the line that says</p><pre class="ql-syntax" spellcheck="false">'forte__active_space_solver' : 'fci'} # &lt;-- specify the reference 
</pre><p>shouldn't it say specify the active space solver instead of specify the reference</p><p>4) Lastly, at the coefficient list specify symmetry that the molecule is ran in and next or above each of the coefficients if possible list what symmetry corresponds to each irrepresentation</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/evangelistalab/forte/pull/246/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>